ambry-rest/src/main/java/com.github.ambry.rest/NettyResponseChannel.java

```diff
@@ -400,7 +400,7 @@ private boolean maybeWriteResponseMetadata(HttpResponse responseMetadata,
         fullHttpResponse.headers().set(responseMetadata.headers());
         responseMetadata = fullHttpResponse;
       }
-      logger.trace("Sending response with status {} on channel {}", responseMetadata.status(), ctx.channel());
+      logger.info("Sending response with status {} on channel {}", responseMetadata.status(), ctx.channel());
```

Comment:
This seems like a bad idea, maybe it was for testing?

Reply:
Good catch, will change it back to trace.